packages/docs-site/README.md

```diff
@@ -29,6 +26,7 @@ All commands are run from the root of the project, from a terminal:
 | `pnpm run preview`         | Preview your build locally, before deploying     |
 | `pnpm run astro ...`       | Run CLI commands like `astro add`, `astro check` |
 | `pnpm run astro -- --help` | Get help using the Astro CLI                     |
+| `cd ../../ && pnpm build:libs`          | Run at root to build the libraries for the docs site            |
```

Comment:
can also do `pnpm run -w build:libs` (runs at the workspace level)

Reply:
```suggestion
| `pnpm run -w build:libs`          | Run at root to build the libraries for the docs site            |
```